gnomad_qc/v5/sample_qc/relatedness.py

```diff
@@ -81,6 +82,21 @@ def print_cuking_command(
     print(textwrap.dedent(cuking_command))
 
 
+def convert_cuking_output_to_ht(cuking_output_path: str) -> hl.Table:
+    """
+    Convert cuKING output Parquet files to a Hail Table.
+
```

Comment:
```suggestion

    .. note ::
    
        We cannot import this function from the cuKING script, which is why it is duplicated here.
        
```
or some other rewording (also the formatting may not be 100% correct)